.htaccess

```diff
@@ -80,7 +80,8 @@ deny from env=stayout
     # <IfModule headers_module>
     #   Header set Strict-Transport-Security "max-age=31536000; includeSubDomains" env=HTTPS
     # </IfModule>
-    # RewriteCond %{HTTPS} !=on
+    # RewriteCond %{HTTP:X-Forwarded-Proto} !https
+    # RewriteCond %{HTTPS} off
```

Comment:
why did you change this one as well?
I don't remember the exact reason but I seem to remember there was a reason why we didn't have off but !=on maybe on some servers that is was configured as 0